### PR TITLE
Features/8076

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
## Status
**Ready**

## Fixes or Implements [AB#8076](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8076)
- [x] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
Adds a print button to timeline menu.
Hide elements not supposed to be printed.

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

### UI Changes
YES

### Browsers Tested
- [x] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <b-navbar toggleable="lg" type="dark">
+  <b-navbar toggleable="md" type="dark">
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue]: why this change

Comment (tiagograf-AOT) [reply Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue]: makes the hamburguer menu show up only in medium size, prevents it from happening while printing the page, and it does fit in the medium view.
